phase2/addon/lib/controller.js

```diff
@@ -655,6 +656,12 @@ const deliverer = {
 
     let rejectDelivery = false;
 
+    if (dh.buttonInited() && (!dh.buttonPlacement() || dh.buttonPlacement() == 'addon-bar')){
+      console.log("delivery rejected due to button replacement");
+      statsEvent("button-replacement-reject", {type: "delivery"});
```

Comment:
this is just so that it's consistent with other delivery rejection labels, e.g. passive-mode-reject, private-reject, inactive-reject, ...